microsoft-edge/toc.yml

```diff
@@ -1336,6 +1336,11 @@
           href: web-platform/password-reveal.md
           displayName: 
 
+        - name: Operating System Regional Data Display in Microsoft Edge
```

Comment:
```suggestion
        - name: Operating System Regional Data Display
```

Reply:
@travisleithead - does the title make sense if remove "edge", in context of the doc-set?